From 113f074d395657c6903c5666bc071c277667eb46 Mon Sep 17 00:00:00 2001
From: Manjeet Rulhania <mrulhania@google.com>
Date: Thu, 30 Jun 2022 22:09:09 +0000
Subject: [PATCH] Remove package name from SafetyNet logs

Bug: 213323615
Test: AppSecurityTests
Change-Id: I993832e148636f1795ffe393c6dc74a08b9442f8
Merged-In: I8f823487ca16861a35135cfc3383fa2ce8258017
Merged-In: I4b61d13256ce0bfb8fc9d21db52ee78ce2097f14
(cherry picked from commit bbe2a118277d9b225a272c53e509fdeb2fe0a993)
Merged-In: I993832e148636f1795ffe393c6dc74a08b9442f8
---
 .../content/pm/parsing/component/ParsedPermissionUtils.java    | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java b/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
index 9a6742c68ab..fa7cfceb1a4 100644
--- a/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
+++ b/core/java/android/content/pm/parsing/component/ParsedPermissionUtils.java
@@ -247,8 +247,7 @@ public class ParsedPermissionUtils {
                 final ParsedPermission perm = checkDuplicatePerm.get(name);
                 if (isMalformedDuplicate(parsedPermission, perm)) {
                     // Fix for b/213323615
-                    EventLog.writeEvent(0x534e4554, "213323615",
-                            "The package " + pkg.getPackageName() + " seems malicious");
+                    EventLog.writeEvent(0x534e4554, "213323615");
                     return true;
                 }
                 checkDuplicatePerm.put(name, parsedPermission);
-- 
2.37.1.359.gd136c6c3e2-goog

